### DISCUSSION TITLE
I am getting this issue in server side

### DISCUSSION THREAD

**Discussion by om-666** [Sunbird-Ed/Community#577] [Bugs] [answered]
<img width="960" alt="image" src="https://github.com/Sunbird-Ed/Community/assets/104663400/24582633-ff01-4a53-9a10-dda757f76943">
![image](https://github.com/Sunbird-Ed/Community/assets/104663400/86740ef1-30cd-46b6-8a93-b35e2b3ec6b5)
![image](https://github.com/Sunbird-Ed/Community/assets/104663400/f82fbdaf-25e5-49bd-ab06-0e98e10fe124)
 I am getting the this issue in server side after running this command npm run server anyone please help me out to resolve this issue

**Comment by chitranshu-keshav** [comment]
@vinukumar-vs @rajeevsathish Can you please get back on this

**Comment by rajeevsathish** [comment]
@om-666 this is ok as this plugin is looking for a db which is not there in the local and will not effect the local portal setup because of this
cc: @chitranshu-keshav @vinukumar-vs

**Reply by om-666** [reply]
What  is the solution for this.

**Reply by om-666** [reply]
Can u please tell me

**Comment by rajeevsathish** [comment, marked as answer]
@om-666  
as this is server side code you do not need a solution for this to run the Portal instance in local.

**Reply by om-666** [reply]
issues  resolved
